code/modules/trade/datums/trade_stations_presets/1-common/hellcat.dm

```diff
@@ -88,7 +88,7 @@
 		"Xanorath Smugged Gear" = list(
 			/obj/item/clothing/suit/space/void/merc/xanorath,
 			/obj/item/melee/energy/sword/pirate,
-			/obj/item/gun/projectile/gyropistol,
+			/obj/item/gun/projectile/gyropistol = good_data("Man-Opener", list(0, 2), 28000), //holster sized 'i win' gun, keep this expensive and rare.
```

Comment:
wrong name

Reply:
The joke is that Voidwolves call it a man-opener even though that's not strictly its name